src/WordEmbeddingModel.scala

```diff
@@ -1,130 +0,0 @@
-//package cc.factorie.app.nlp.embeddings
-import cc.factorie.model.{Parameters, Weights}
-import cc.factorie.optimize.{Trainer, AdaGradRDA}
-import cc.factorie.la.DenseTensor1
-import cc.factorie.util.Threading
-import java.io.{File, PrintWriter, OutputStreamWriter, FileOutputStream, FileInputStream, BufferedOutputStream}
-import java.util.zip.{GZIPOutputStream, GZIPInputStream}
-
-
-abstract class WordEmbeddingModel(val opts: EmbeddingOpts) extends Parameters {
```

Comment:
Interface for Skip-Gram based models (in here there is a vanilla SkipGram with neg sampling). Also contains persistence storage method, vocab building method, and `learnEmbeddings` method which seems to be a wrapper for a model's threaded `processExamples` implementation.